packages/session-replay-browser/src/session-replay.ts

```diff
@@ -318,20 +334,24 @@ export class SessionReplay implements AmplitudeSessionReplay {
     return maskSelector as unknown as string;
   }
 
-  recordEvents() {
-    const shouldRecord = this.getShouldRecord();
+  captureEventsIfShould() {
```

Comment:
Can we instead call this `captureEvents` or add a doc string here, since at first glance it's not immediately obvious what if should is in this context.

Reply:
Yeah sounds good, agreed the `ifShould` is confusing to read